quizzes/midterm.md

```diff
@@ -51,14 +55,20 @@
 
 	ant_eater
 	```
+	your not passing it a variable so nil or default value but i don't see how it would run 
 
 1. Name one thing you would fix with the arguments of this method. (see below)
```

Comment:
5/10 - Was looking for: Move the optional arguments to the end of the args list.